Loenn/triggers/spawn_steam_wall.lua

```diff
@@ -0,0 +1,20 @@
+local spawnSteamWall = {}
+
+spawnSteamWall.name = "FactoryHelper/SpawnSteamWallTrigger"
+spawnSteamWall.fieldInformation = {
+    speed = {
+        minimumValue = 0.0
```

Comment:
I tested this out with a negative value and it was totally functional, I feel like we might as well allow it lol (also applies to the steam wall speed trigger)

Reply:
Thanks for testing, was unsure, so I added the minimum limit going off the field's name.